### DISCUSSION TITLE
🎉

### DISCUSSION THREAD

**Discussion by IanNorris** [simple-login/app#882] [Kind words ❤️]
I started using the service this week, and I'm pretty sold already, despite a few niggles (see the new discussions I just opened).

My primary reservation has been around security (building secure software is hard) - I work for a major tech company and my role involves software security, so the fact that you're currently undergoing a security review AND that you've been bought by Proton is music to my ears. It also reaffirms my choice to trust SimpleLogin with what is essentially the most component in my security posture.

I'm looking forward to seeing what great things you can achieve under Proton, and hopefully the additional exposure that brings.

**Comment by nguyenkims** [comment]
@IanNorris thanks for the kind words! We have actually done a security audit recently and can't wait to share the result with the community!
